en/controllers.rst

```diff
@@ -463,6 +449,25 @@ the controller's default one::
         )
         ->all();
 
+.. php:method:: fetchModel(string $alias, array $config = [])
+
+The ``fetchModel()`` method is useful to load non ORM models or ORM tables that
+are not the controller's default::
```

Comment:
> or ORM tables that are not the controller's default

This is the same definition as fetchTable() which is confusing. We should describe fetchModel() for just one use case.

Reply:
Well "non ORM models" is the different bit here. That method can fetch any type of model including tables, which is why you get if you don't specify the 2nd argument.